plugins/statistics.koplugin/main.lua

```diff
@@ -17,6 +17,14 @@ local tableutil = require("tableutil")
 
 local statistics_dir = DataStorage:getDataDir() .. "/statistics/"
 local history_dir = DataStorage:getHistoryDir()
+local history_file = joinPath(DataStorage:getDataDir(), "history.lua")
+
+local ReadHistoryStat = { 
```

Comment:
I don't see any other reference to this variable except in readHistoryFile, is it still needed for this change?